Vagrantfile

```
@@ -0,0 +1,27 @@
+Vagrant.configure("2") do |config|
+  config.vm.box = "hashicorp/bionic64"
+  config.vm.provider "virtualbox"
+  config.vm.network "forwarded_port", guest: 5000, host: 5000
+  config.vm.provision "shell", privileged: false, inline: <<-SHELL
+    sudo apt-get update
+    sudo apt-get install -y build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev xz-utils tk-dev libffi-dev liblzma-dev python-openssl git
+    git clone https://github.com/pyenv/pyenv.git ~/.pyenv
+    echo 'export PYENV_ROOT=\"$HOME/.pyenv\"' >> ~/.profile
+    echo 'export PATH=\"$PYENV_ROOT/bin:$PATH\"' >> ~/.profile
+    echo -e 'if command -v pyenv 1>/dev/null 2>&1; then\n  eval  \"$(pyenv init -)\"\nfi' >> ~/.profile
+    source ~/.profile
+    $HOME/.pyenv/bin/pyenv install 3.8.5
+    $HOME/.pyenv/bin/pyenv global 3.8.5
```

Your source line above should mean that pyenv is available directly on these lines:
```suggestion
    pyenv install 3.8.5
    pyenv global 3.8.5
```